lib/rmt/downloader.rb

```diff
@@ -72,7 +58,7 @@ def process_queue
       begin
         make_request(remote_file, local_file, request_fiber, queue_item[:checksum_type], queue_item[:checksum])
       rescue RMT::Downloader::Exception => e
-        @logger.info("E #{File.basename(local_file)} - #{e}")
+        @logger.warn("× #{File.basename(local_file)} - #{e}")
```

Comment:
What kind of char is `×`? Is it supported on _all_ terminals?
Also, is it appropriate to use for logging? Does it have any meaning? Looks weird to me.